vespa_analysis/vespa_msa.py

```diff
@@ -0,0 +1,67 @@
+import os
+import argparse
+from Bio import SeqIO
+import pandas as pd
+import numpy as np
+
+
+def sav_stats(gen_id,gen_aln,vespa_map_out):
+    sav_vector = []
+    c = 0
+    df = pd.read_csv(vespa_map_out)
+    seq = gen_aln
+
+    for j in seq:
+        if j == '_':
+            sav_vector.append(0)
+        else:
+            c=c+1
+            if c-1 in list(df['Gen Seq Index'].values):
+                sav_vector.append(list(df['Score'].values)[list(df['Gen Seq Index'].values).index(c-1)]*10)
+            else:
+                sav_vector.append(0)
+    return sav_vector
+
+def cons_stats(ref_id,ref_aln,cons_class):
+    cons_class = cons_class.split(',')
+    ref_seq = ref_aln
+    cons_vector = []
+    c = 0
+    for j in ref_seq:
+        if j == '_':
+            cons_vector.append(0)
+        else:
+            c=c+1
+            cons_vector.append(int(cons_class[c-1]))
+    return cons_vector
+
+parser = argparse.ArgumentParser(description='Computes SAV and Conservation scores statistics')
+parser.add_argument('--vespa_map_out', type=str, action='store', help='FULL Path of vespa mapped score output csv file',required=True)
+parser.add_argument('--ref_id', type=str, action='store', help='reference sequence ID',required=True)
+parser.add_argument('--ref_aln', type=str, action='store', help='MSA reference sequence',required=True)
+parser.add_argument('--cons_class', type=str, action='store', help='Conservation classes of reference sequence',required=True)
+parser.add_argument('--gen_id', type=str, action='store', help='generated sequence ID',required=True)
+parser.add_argument('--gen_aln', type=str, action='store', help='MSA generated sequence',required=True)
+args = parser.parse_args()
+
+
+## Inputs ##
+mapped_scores = args.vespa_map_out
```

Comment:
you can return all of these inputs from the argument parser function as I described in the comment in line 38.